srcpkgs/xst/template

```diff
@@ -21,7 +21,6 @@ do_install() {
 	vinstall st.info 644 usr/share/terminfo/x xst.terminfo
 	vdoc README
 	vdoc FAQ
-	vsed -i 's/st-256color/xst-256color/' .Xresources
```

Comment:
I believe this change was made by @ndowens , iirc? Why remove it?

Reply:
no, 271d64c5de1526e5cbfbf474ef56149ab2274193. @ericonr the second commit seems to clarify it already (`Replacing st-256color to xst-256color became unnecessary since it has been fixed in the upstream code.`)
@dotnetfox please squash the commits so you offer one commit per package

Reply:
Ah, thanks.

Reply:
> I believe this change was made by @ndowens , iirc? Why remove it?

Nah it was already in previous version, but I left in in case removing would have conflicted with st